.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "printWidth": 120,
+  "singleQuote": true,
+  "bracketSpacing": false,
+  "trailingComma": "es5"
+}
```

Comment:
I’m not familiar with the eslint/prettier configuration. But is it maybe possible to move it into the `package.json` files to avoid more config files?

Alternatively maybe moving them into a `.github` folder if they support it?

Reply:
I don't think `.github` is supported. But `package.json` is, so I'm gonna go with that.
I could have used the default setting of `prettier` (no file required) I went with those settings as they would limit greatly the reformatting on other repo.

I'll try to move as many dotfiles conf into `package.json`

Reply:
awesome, thank you very much!

Reply:
Done. I also moved `.eslintrc` to `package.json`